files.sh

```diff
@@ -2,7 +2,7 @@
 
 fn nash_complete_paths(parts, line, pos) {
 	partsz   <= len($parts)
-	last     <= -expr $partsz - 1
+	last     <= -echo $partsz - 1 | bc
```

Comment:
Great @vitorarins =)

But `multiple-returns` branch still need some work as pointed out by @katcipis review. Maybe makes sense to make a hot fix in master for now.